src/gwt/src/org/rstudio/studio/client/workbench/commands/Commands.cmd.xml

```diff
@@ -839,6 +837,7 @@ well as menu structures (for main menu and popup menus).
          <shortcut refid="helpSearch" value="Ctrl+Alt+F1"/>
          <shortcut refid="helpBack" value="Shift+Alt+F2"/>
          <shortcut refid="helpForward" value="Shift+Alt+F3"/>
+         <shortcut refid="showCommandPalette" value="Ctrl+Shift+P"/>
```

Comment:
FireFox on Windows launches a new Private Browsing window with this shortcut. Worked on everything else I tried (well, I didn't try Linux desktop browsers, and IE11 is very slow with this feature).